persistence/trees/trees.go

```diff
@@ -341,3 +350,23 @@ func (t *treeStore) updateFlagsTree(flags []*coreTypes.Flag) error {
 
 	return nil
 }
+
+func (t *treeStore) updateIBCTree(keys, values [][]byte) error {
+	if len(keys) != len(values) {
+		return fmt.Errorf("keys and values must be the same length")
+	}
+	for i := 0; i < len(keys); i++ {
+		key := keys[i]
+		value := values[i]
+		if value == nil {
+			if err := t.merkleTrees[IBCTreeName].tree.Delete(key); err != nil {
+				return err
+			}
+		} else {
+			if err := t.merkleTrees[IBCTreeName].tree.Update(key, value); err != nil {
+				return err
+			}
+		}
```

Comment:
i just think it's easier on the eys

```suggestion
		var err error
		if value == nil {
			err = t.merkleTrees[IBCTreeName].tree.Delete(key)
		} else {
			err := t.merkleTrees[IBCTreeName].tree.Update(key, value)
		}
		return err
```

Reply:
Noted, maybe just use an `else if` instead?

Reply:
changed to use a continue and removed 2nd branch